cmd/ris-mirror/rismirror/vrf.go

```diff
@@ -1,19 +1,24 @@
 package rismirror
 
 import (
-	"github.com/bio-routing/bio-rd/routingtable/locRIB"
+	"github.com/bio-routing/bio-rd/risclient"
 	"github.com/bio-routing/bio-rd/routingtable/mergedlocrib"
+	"github.com/bio-routing/bio-rd/routingtable/vrf"
 )
 
 type _vrf struct {
+	vrf         *vrf.VRF
 	ipv4Unicast *mergedlocrib.MergedLocRIB
 	ipv6Unicast *mergedlocrib.MergedLocRIB
+	Clients     []*risclient.RISClient
 }
 
-func newVRF(locRIBIPv4Unicast, locRIBIPv6Unicast *locRIB.LocRIB) *_vrf {
+func newVRF(vrf *vrf.VRF) *_vrf {
 	return &_vrf{
-		ipv4Unicast: mergedlocrib.New(locRIBIPv4Unicast),
-		ipv6Unicast: mergedlocrib.New(locRIBIPv6Unicast),
+		vrf:         vrf,
+		ipv4Unicast: mergedlocrib.New(vrf.IPv4UnicastRIB()),
+		ipv6Unicast: mergedlocrib.New(vrf.IPv6UnicastRIB()),
+		Clients:     make([]*risclient.RISClient, 2),
```

Comment:
Here is the issue! This creates to nil entries in the slice. I guess you want to set capacity, not length.
`make([]*risclient.RISClient, 0, 2)`

Reply:
Updated it